src/time.rs

```diff
@@ -6,11 +6,10 @@
 
 use chrono::offset::Utc;
 use std::{thread, time};
-use std::sync::Arc;
 use std::sync::atomic::{AtomicUsize, Ordering};
 
 lazy_static! {
```

Comment:
Why does this one remain?

Reply:
Ah! For a thing to be static all the initialization code has to be callable at compile-time. Can't read the time at compilation time, so we gotta initialize lazily.